src/patterns/equality-information/explainer-screen/index.njk

```diff
@@ -0,0 +1,52 @@
+---
+title: Explainer screen for equality information questions
+layout: layout-example.njk
+---
+
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% from "govuk/components/details/macro.njk" import govukDetails %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% set hintHtml %}
+  <p class="govuk-!-margin-top-0">These questions are optional. [Add a couple of sentences explaining why you’re asking the questions and what you’ll do with the information].</p>
+  <p>Your answers won’t affect your application.</p>
+{% endset -%}
+
+<h1 class="govuk-heading-l">We have received your application</h1>
+
+<p class="govuk-body">Before you finish using the service, we’d like to ask some equality questions.</p>
+
+<form action="/form-handler" method="post" novalidate>
```

Comment:
This ends up nested inside the `<form>` tag from the layout, which causes an [HTML validation error](https://validator.w3.org/nu/?doc=https%3A%2F%2Fdeploy-preview-1581--govuk-design-system-preview.netlify.app%2Fpatterns%2Fequality-information%2Fexplainer-screen%2Findex.html):

> "Saw a form start tag, but there was already an active form element. Nested forms are not allowed. Ignoring the tag."

If you look in dev tools there's no trace of it, which I didn't know was a thing! TIL.

Can see the value in providing it as part of the example, but none of the other examples include it and there'd be no harm if the `<form>` tag wrapped the whole thing, so wondering if the simplest thing is to just remove it?